examples/my_first_infertrade_strategy.py

```diff
@@ -23,6 +23,9 @@
 import pandas as pd
 import matplotlib.pyplot as plt
 from infertrade.utilities.performance import calculate_portfolio_performance_python
+import matplotlib
+
+matplotlib.use('TKAgg')
```

Comment:
What is this needed for?

Reply:
I think it os related Tkagg seems like a package that is not provided by certain ubuntu versions. I guess we can either install sudo apt-get install tk-dev libpng-dev libffi-dev dvipng texlive-latex-base manually on the semaphore and then build the project and we won't need to add the above statement.

https://stackoverflow.com/questions/12933335/tkagg-backend-problems?rq=1